README.md

```diff
@@ -90,68 +61,125 @@ DEBUG_PORT=5005
 SPRING_LOCAL_PORT=8081
 SPRING_DOCKER_PORT=8080
 ```
-
-3. **Create `docker-compose.yaml` in your root directory**
-
-```yaml
-version: "3.8"
-
-services:
-  mysqldb:
-    image: mysql:latest
-    restart: always
-    env_file: ./.env
-    environment:
-      MYSQL_DATABASE: ${MYSQLDB_DATABASE}
-      MYSQL_ROOT_PASSWORD: ${MYSQLDB_ROOT_PASSWORD}
-    ports:
-      - "${MYSQLDB_LOCAL_PORT}:${MYSQLDB_PORT}"
-    healthcheck:
-      test: ["CMD-SHELL", "mysqladmin ping -h 127.0.0.1 -u${MYSQLDB_USER} -p${MYSQLDB_ROOT_PASSWORD}"]
-      interval: 30s
-      timeout: 10s
-      retries: 3
-
-  app:
-    depends_on:
-      mysqldb:
-        condition: service_healthy
-    restart: on-failure
-    image: book-shop
-    build: .
-    env_file: ./.env
-    ports:
-      - ${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}
-      - ${DEBUG_PORT}:${DEBUG_PORT}
-    environment:
-      SPRING_APPLICATION_JSON: '{
-        "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_PORT}/${MYSQLDB_DATABASE}",
-        "spring.datasource.username":"${MYSQLDB_USER}",
-        "spring.datasource.password":"${MYSQLDB_ROOT_PASSWORD}",
-        "spring.jpa.properties.hibernate.dialect":"org.hibernate.dialect.MySQLDialect"
-      }'
-      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
-```
-
-4. **Build the project**
+3. **Build the project**
 ```bash
 ./mvn clean package
 ```
-5. **Build the images**
-``` bash 
-docker-compose build
-```
-6. **Run the Docker compose**
+4. **Run the Application**
+  - **Using Docker Compose**
+   ```bash
+   docker-compose up --build
+   ```
+  - **Using Maven**
+   ```bash
+   mvn spring-boot:run
+   ```
+7. **Run the Docker compose**
 ```bash
 docker-compose up
```

Comment:
you do not need this step, you already have it above, just with the `--build`, which means that we also want to build images before starting containers